bears/js/JSONFormatBear.py

```diff
@@ -37,10 +38,16 @@ def run(self, filename, file,
             json_content = json.loads(''.join(file),
                                       object_pairs_hook=OrderedDict)
         except JSONDecodeError as err:
+            err_content = str(err).split(':')
+            err_msg = err_content[0]+'.'
+            position = findall(r'\d+', err_content[1])
             yield Result.from_values(
                 self,
-                'This file does not contain parsable JSON. ' + repr(str(err)),
-                file=filename)
+                'This file does not contain parsable JSON. ' +
+                repr(str(err_msg)),
```

Comment:
though you haven't introduced it, if you want you can push another commit which corrects this to just `err_msg` instead of `repr(str(err_msg))` (as `err_msg` is already a string, `str` has no effect + `repr` also renders the string like it has to look in a python file when you write this string, just the message itself is okay).